src/database/DataTypes/Report.js

```diff
@@ -30,7 +31,7 @@ export class Report extends Realm.Object {
    * @param  {dataObject}  dataObject
    */
   set data(dataObject) {
-    this._data = JSON.stringify(dataObject);
+    this._data = checkIsObject(dataObject) ? JSON.stringify(dataObject) : {};
```

Comment:
oops, you'll still need to stringify the `{}`

Reply:
Thanks! Done here 219774d